ptr.py

```diff
@@ -612,10 +612,13 @@ async def _test_steps_runner(  # pylint: disable=R0914
 
                 # If we're running tests and we want warnings to be errors
                 step_env = env
-                if a_step.step_name == StepName.tests_run and error_on_warnings:
+                if a_step.step_name in [StepName.tests_run, StepName.pyre_run]:
                     step_env = env.copy()
+                    step_env["PYTHONPATH"] = getcwd()
+
+                if a_step.step_name == StepName.tests_run and error_on_warnings:
                     step_env["PYTHONWARNINGS"] = "error"
-                    LOG.debug(f"Setting PYTHONWARNINGS to error: {step_env}")
+                    LOG.debug("Setting PYTHONWARNINGS to error")
```

Comment:
To spammy? I remember it being helpful working things out.

Reply:
If it is useful, it should occur if the env is modified, which would be a new `if a_step.step_name in [StepName.tests_run, StepName.pyre_run]:` branch below this, or some better env-modified tracking flag used to determine whether to log it.

To answer your other question, no I am not needing a release.  This is just prep work so that the openSUSE rpm can be simplified next release, and other distros benefit from it.